src/registrar/views/portfolios.py

```diff
@@ -341,6 +407,91 @@ def get(self, request, pk):
             },
         )
 
+    def post(self, request, pk):
+        """
+        Handles adding and removing domains for a portfolio invitee.
+        """
+        added_domains = request.POST.get("added_domains")
+        removed_domains = request.POST.get("removed_domains")
+        portfolio_invitation = get_object_or_404(PortfolioInvitation, pk=pk)
+        email = portfolio_invitation.email
+
+        added_domain_ids = self._parse_domain_ids(added_domains, "added domains")
+        if added_domain_ids is None:
+            return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+
+        removed_domain_ids = self._parse_domain_ids(removed_domains, "removed domains")
+        if removed_domain_ids is None:
+            return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+
+        if added_domain_ids or removed_domain_ids:
+            try:
+                self._process_added_domains(added_domain_ids, email)
+                self._process_removed_domains(removed_domain_ids, email)
+                messages.success(request, "The domain assignment changes have been saved.")
+                return redirect(reverse("invitedmember-domains", kwargs={"pk": pk}))
+            except IntegrityError:
+                messages.error(request, "A database error occurred while saving changes.")
+                return redirect(reverse("invitedmember-domains-edit", kwargs={"pk": pk}))
+            except Exception as e:
+                messages.error(request, f"An unexpected error occurred: {str(e)}")
+                return redirect(reverse("invitedmember-domains-edit", kwargs={"pk": pk}))
+        else:
+            messages.info(request, "No changes detected.")
```

Comment:
**possibly blocking** I would prefer that we just show this if a user clicks the review button without having clicked anything. It seems very odd they can get get to the review page with no changes. Would this be too much of an addition to add in?

Reply:
I think this is another question for design. We typically do not limit the interaction with submit buttons, but rather let the user hit submit and respond to whatever selection/ errors / no selections they have made. Our implementation follows that pattern. FWIW, I think it's the right pattern.

Reply:
Agree that it follows our current pattern, which doesn't limit submission if no changes were made. I'd suggest keeping it consistent for now, but maybe this is something we could circle back to later - across registrar forms. cc: @gabydisarli for input

Reply:
Sounds good, resolving this convo